fieldservice_agreement/models/agreement_serviceprofile.py

```diff
@@ -0,0 +1,11 @@
+# Copyright (C) 2018 - TODAY, Pavlov Media
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+
+from odoo import fields, models
+
+
+class AgreementServiceProfile(models.Model):
+    _inherit = 'agreement.serviceprofile'
+
+    notes = fields.Text(string="Notes")
```

Comment:
@osi-scampbell Should be moved to `agreement_serviceprofile`.

Reply:
That was my first thought but I figured if OCA took it out then they wouldn't want it put back in. I'll move it right away.